.prettierignore

```diff
@@ -2,4 +2,5 @@
 build/
 coverage/ 
 storybook-static/
-db.json/
\ No newline at end of file
+./db.json/
+client/db.json
```

Comment:
Why do you have two db.json's

Reply:
We weren't sure how to add files to the .prettierignore and tried some cases which we forgot to delete later - thanks for the tip!